products/models.py

```diff
@@ -102,3 +103,16 @@ class Meta:
 
     def __str__(self):
         return f'{self.banner_name}'
+
+
+class Cart(models.Model):
+    """this model is useful to add products into cart """
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
```

Comment:
@Yshah112 can we use models.PROTECT for user and product fields?

cc: @lalitvasoya

Reply:
done